internal/objects/job/job.go

```diff
@@ -139,6 +139,11 @@ func DesiredJob(contour *operatorv1alpha1.Contour, image string) *batchv1.Job {
 		SchedulerName:                 "default-scheduler",
 		TerminationGracePeriodSeconds: pointer.Int64Ptr(int64(30)),
 	}
+
```

Comment:
I know this deviates from upstream. Could we at some stage revert back to align with the upstream operator or do you think we should continue ?

Reply:
This change should be also merged into upstream. I will mention it in the PR which is pending upstream.

Reply:
But for now this should be merged.